PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugSettingsEditor.java

```diff
@@ -42,6 +58,17 @@ protected void applyEditorTo(@NotNull RemoteDebugRunConfiguration remoteDebugRun
     @NotNull
     @Override
     protected JComponent createEditor() {
-        return new JPanel();
+
+        Project project = myRunConfiguration.getProject();
+        List<IClusterDetail> cachedClusterDetails = ClusterManagerEx.getInstance().getClusterDetailsWithoutAsync(true, project);
```

Comment:
It is not on purpose. However, we do want this dialog to pop up promptly. I will open an issue so we can get the no cluster bug fixed. Then this won't be an issue.

---
In reply to: [116899228](https://github.com/Microsoft/azure-tools-for-java/pull/447#discussion_r116899228) [](ancestors = 116899228)